integration_tests/commands/async/hmget_test.go

```diff
@@ -0,0 +1,101 @@
+package async
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestHMGET(t *testing.T) {
+	conn := getLocalConnection()
+	var smallHashKeys []string
+	var smallHashValues []string
+	var bigHashKeys []string
+	var bigHashValues []string
+	// Setup smallHash
+	for i := 0; i < 8; i++ {
+		key := fmt.Sprintf("field%d", i)
+		value := fmt.Sprintf("value%d", i)
+		smallHashKeys = append(smallHashKeys, key)
+		smallHashValues = append(smallHashValues, value)
+		FireCommand(conn, "HSET smallHash "+key+" "+value)
+	}
+	// Setup bigHash
+	for i := 0; i < 1024; i++ {
+		key := fmt.Sprintf("field%d", i)
+		value := fmt.Sprintf("value%d", i)
+		bigHashKeys = append(bigHashKeys, key)
+		bigHashValues = append(bigHashValues, value)
+		FireCommand(conn, "HSET bigHash "+key+" "+value)
+	}
```

Comment:
Why do we need so many values? This looks more like a benchmark here. A test should ideally be small, easy to understand, and easy to debug.

Reply:
Hey, they do look more like benchmarks to me as well. But these were the tests that I ported over from redis. I think we don't really need them. What would you suggest?

Reply:
better to have a few fixed test cases with static inputs and outputs. You can take a look at how most of the other tests in the repository are written.